Something/WebApplication/wwwroot/js/apps/authentication/Authentication.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+
+import { AuthenticationForm } from "./components/AuthenticationForm/AuthenticationForm";
+import { Splash } from "./components/Splash/Splash";
+
+import "./../Global.scss";
+import "./Authentication.scss";
+import { IAuthenticationProps } from "./types/IAuthenticationProps";
+
+export default class Login extends React.Component<{}, {}> {
+    public render(): JSX.Element {
+
+        const params:IAuthenticationProps = (window as any).authenticationPageParams as IAuthenticationProps;
```

Comment:
If you want to make this testable create a window service.

Reply:
Will do.